app/models/metasploit/cache/module/instance.rb

```diff
@@ -303,6 +265,11 @@ class Metasploit::Cache::Module::Instance < ActiveRecord::Base
   #
   #   @return [String]
 
+  # @!method module_class_id
```

Comment:
Is there a reason these aren't `@!attribute`s? Using `@!method` will not create a setter.